omrsigcompat/makefile

```diff
@@ -25,6 +25,18 @@ OBJECTS := omrsig
 OBJECTS := $(addsuffix $(OBJEXT),$(OBJECTS))
 EXPORT_FUNCTIONS_FILE := omrsig.exportlist
 
+ifeq (win,$(OMR_HOST_OS))
```

Comment:
@rwy0717 ,   The problem is I didn't figured out how to generate omr.res in the root OMR.
The situation is that repackaging all source code including omr will create omr.rc but won't generate omr.res, which has to be postponed to the compilation (I tried but there is no way to work out). In such case, it seems the only way to generate omr.res is to directly change the makefile in a specific directory (e.g. in omrsigcompat). 

Let me know if you figure out a better idea to generate omr.res or there is something I am unaware of/missed.